Google.PowerShell/Compute/GceDisk.cs

```diff
@@ -133,6 +133,21 @@ protected void WriteDiskObjects(IEnumerable<Disk> disks)
     /// <para type="description">
     /// Creates a new Google Compute Engine disk object.
     /// </para>
+    /// <example>
+    ///   <code>PS C:\> New-GceDisk "disk-name" -SizeGb 10 -DiskType pd-ssd</code>
+    ///   <para>
+    ///     Creates a new empty 10GB persistant solid state disk named "disk-name" in the default project and
```

Comment:
We wanted to put the descriptions before the code snippets, right?

Also, this includes the ">" character. If we can get away with that instead of &gt;, I'd prefer to keep it in.